dev/ci/ci-quickchick.sh

```diff
@@ -10,6 +10,6 @@ git_download quickchick
 if [ "$DOWNLOAD_ONLY" ]; then exit 0; fi
 
 ( cd "${CI_BUILD_DIR}/quickchick"
-  make
-  make install
+  dune build -p coq-quickchick @install
+  dune install -p coq-quickchick --prefix=$CI_INSTALL_DIR
```

Comment:
```suggestion
  dune build --root . -p coq-quickchick @install
  dune install coq-quickchick --prefix=$CI_INSTALL_DIR
```
IIRC this is how using dune in ci works